idc_index/index.py

```diff
@@ -1314,6 +1343,17 @@ def _format_size(size_MB):
             return f"{round(size_GB, 2)} GB"
         return f"{round(size_MB, 2)} MB"
 
+    @staticmethod
```

Comment:
Let's combine this with the previous function!

Reply:
Did that.